tests/test_trigger.py

```diff
@@ -63,3 +63,36 @@ def test_create_from_user_query(bot_fx):
         "target": "test_nick",
         "message": "Exceedingly smart test message."
     } in bot_fx.sent_messages
+
+def test_trigger_eq(bot_fx):
+    trigger1 = Trigger(bot_fx, ["some", "thing"], ["some thing", "thing"], "test_nick",
+                       "not_a_channel", "test_ident", "test.vhost")
+    trigger2 = Trigger(bot_fx, ["some", "thing"], ["some thing", "thing"], "test_nick",
+                       "not_a_channel", "test_ident", "test.vhost")
+
+    assert trigger1 == trigger1
+    assert trigger2 == trigger2
+
+    assert trigger1 == trigger2
+    assert trigger2 == trigger1
+
+def test_trigger_ne(bot_fx):
+    trigger1 = Trigger(bot_fx, ["some", "thing"], ["some thing", "thing"], "test_nick",
```

Comment:
may i suggest providing `trigger` as a fixture? would reduce duplicity. Failing that you could paramatrize this, so you can test with various inputs rather than a single pair of hardcodes

Reply:
I don't think there's justification for making this a fixture, since I need a few different Trigger objects and they likely wouldn't be used in any other tests.
Parametrized tests wouldn't really increase test coverage much, but definitely wouldn't hurt...